src/helix-ui/elements/HXDropFenceElement.less

```diff
@@ -0,0 +1,10 @@
+@import "./HXElement";
+
+#hxDropFence {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: center;
+  width: 100%;
+}
```

Comment:
```less
#hxDropFence {
  > div {
    max-width: 60ch;
  }
}
```

If we expect a drop fence to _always_ have content, we'll also need to add a `margin-top: 0.5rem;`.

Reply:
added, look at css file -- converted to rem